Website/src/items/content/EdgeWebsite/home/sponsors/Dwell.yml

```diff
@@ -16,6 +16,9 @@ Languages:
   Versions:
   - Version: 1
     Fields:
+    - ID: "001dd393-96c5-490b-924a-b0f25cd9efd8"
+      Hint: __Lock
+      Value: <r />
```

Comment:
I see a lot of serialized items only got an empty lock field addition. It is in an unlocked state so it is fine. It would be better not to have committed those unneeded changes in the first place. We can keep them as is for this PR.